src/Polly.Core/Strategy/IResilienceArguments.cs

```diff
@@ -0,0 +1,12 @@
+namespace Polly.Strategy;
+
+/// <summary>
+/// The recommended shape for arguments used by individual strategies.
+/// </summary>
+public interface IResilienceArguments
```

Comment:
The idea was to enforce some kind of common shape since we now ever argument class will have the `Context` property. By having public interface, we can have unified docs by using `<inheritdoc/>` thingy. 

This interface won't be used anywhere directly, just as a marker.